deploy/charts/external-secrets/templates/webhook-service.yaml

```diff
@@ -7,20 +7,18 @@ metadata:
   labels:
     {{- include "external-secrets-webhook.labels" . | nindent 4 }}
     external-secrets.io/component: webhook
-  {{- if .Values.webhook.metrics.service.enabled }}
-  {{- with .Values.webhook.metrics.service.annotations }}
-  annotations:
-    {{- toYaml . | nindent 4 }}
-  {{- end }}
-  {{- end }}
+  {{- include "external-secrets-webhook.annotations" . | nindent 2 }}
```

Comment:
Isn't this slightly different now? Like, the included template block has more if statements for the annotation to be enabled than just plain `service.enabled`.

this 
```yaml
{{- if or .Values.webhook.service.annotations (and .Values.webhook.metrics.service.enabled .Values.webhook.metrics.service.annotations) -}}
```

vs this
```yaml
  {{- if .Values.webhook.metrics.service.enabled }}
```

Reply:
The removed if-statement from here has been moved into the helper.

I think all the required logic is in that helper.  It shows `annotations:` (and possible content) only when `service.annotations or (metrics...enabled and metrics...annotations)`.

Am I misunderstanding your comment?

Reply:
Want me to add some tests to verify? I could add those if you like.

Reply:
The comment was that in the helper the if statement chnaged compared to the simple one now it checked if annotation where there or something like that.

Reply:
Exactly!